package.json

```diff
@@ -20,8 +20,8 @@
   "homepage": "https://github.com/asyncapi/html-template#readme",
   "scripts": {
     "lint": "echo \"No linter specified yet\"",
-    "develop": "npm run generate:assets && ag https://raw.githubusercontent.com/asyncapi/generator/v1.1.5/test/docs/dummy.yml ./ -o test/output --force-write --watch-template",
-    "//develop:install": "Force ag to install your local html-template source code",
+    "test:generator": "node ./node_modules/@asyncapi/generator/cli.js https://raw.githubusercontent.com/asyncapi/generator/v1.1.5/test/docs/dummy.yml ./ -o test/output --force-write",
```

Comment:
we should use `@asyncapi/cli` and generator cli will be deprecated, removed and is no longer even promoted in docs

also since cli will be in devDependencies, afaik you should be able to run `asyncapi` without pointing to `node-modules`

Reply:
yeah, I know you added generator, as cli did not support it yet, but yeah, now it does :)

Reply:
also, since you added 2 test files, v2 and v3 - can we use them here instead of remote one? both?

Reply:
I can revert this change sure 🙂